web-animations-1/Overview.bs

```diff
@@ -628,6 +628,21 @@ a document's load event.
 A <a>timeline</a> is considered to be <dfn lt="inactive timeline">inactive</dfn>
 when its <a>time value</a> is <a>unresolved</a>.
 
+A [=timeline=] may be in one of four possible <dfn>phase</dfn>s:
+
+* <dfn lt="inactive phase">inactive</dfn>
+* <dfn lt="before phase">before</dfn>
+* <dfn lt="active phase">active</dfn>
+* <dfn lt="after phase">after</dfn>
+
+Calculating which phase a timeline is in is handled by each type of [=timeline=]
```

Comment:
I don't think we need this sentence. But we probably do need to say that document timelines are always active.

Reply:
According to the spec, there are cases where a document timeline can be inactive:
"A document timeline that is associated with a Document which is not an active document is also considered to be inactive."
I have removed the unnecessary sentence though. I will update the spec in regards to Document Timeline to explain how it determines what phase it is in.